assets/dev/js/editor/document/globals/commands/disable.js

```diff
@@ -34,7 +34,15 @@ export class Disable extends DisableEnable {
 				container.globals.set( setting, '' )
 			);
 
-			container.settings.set( '__globals__', container.globals.toJSON() );
+			$e.internal( 'document/elements/set-settings', {
+				container,
+				settings: {
+					__globals__: container.globals.toJSON(),
+				},
+				options: {
+					render: false,
+				},
```

Comment:
How it worked until now?

Reply:
Next line `container.renderUI()` you want to add it into `set-settings` as options?

Reply:
yes.

Reply:
Handled.